hkmc2/shared/src/main/scala/hkmc2/codegen/Lifter.scala

```diff
@@ -234,55 +236,38 @@ class Lifter(handlerPaths: Opt[HandlerPaths])(using State, Raise):
     val FreeVars(_, cap) = ctx.usedLocals(f.sym)
 
     val fresh = FreshInt()
-
-    val varsMap = cap.map: s =>
```

Comment:
The creation and subsequent repeated querying of this intermediate map was entirely gratuitious and resulted in needless overhead.